### PR TITLE
Replace () with Msg containing NoOp value

### PR DESCRIPTION
**Pull request by xypnox**

### CODE REVIEW

File: frontend/Pages/Index.elm [hunk 1]
```diff
@@ -11,7 +11,7 @@ main =
     R.app app
 
 
-app : R.App Config Config ()
+app : R.App Config Config Msg
```
Comment (amitu) [inline frontend/Pages/Index.elm]: Lets also use a `Model` here, instead of reusing `Config` as `Model`.

File: frontend/Pages/Index.elm [hunk 2]
```diff
@@ -11,9 +11,14 @@ main =
...
-app : R.App Config Config Msg
+init : R.In -> Config -> ( Model, Cmd (R.Msg Msg) )
+init _ c =
+    ( { config = c }, Cmd.none )
+
+
+app : R.App Config Model Msg
 app =
-    R.App config R.init0 R.update0 R.sub0 document
+    R.App config init R.update0 R.sub0 document
```
Comment (amitu) [inline frontend/Pages/Index.elm]: Lets remove ` R.update0 R.sub0` too.

Comment (xypnox) [reply frontend/Pages/Index.elm]: I have added the update and sub too. I have also switched their type definitions from type variable to the Model type used in the file.

File: frontend/Pages/Index.elm [hunk 3]
```diff
@@ -35,6 +21,21 @@ type Msg
...
 
 
+init : R.In -> Config -> ( Model, Cmd (R.Msg Msg) )
+init _ c =
+    ( { config = c }, Cmd.none )
+
+
+update : R.In -> Msg -> Model -> ( Model, Cmd (R.Msg Msg) )
+update _ _ model =
+    ( model, Cmd.none )
```
Comment (amitu) [inline frontend/Pages/Index.elm]: Rename the variable `model` to `m`.

**Pull request merged**